cypress.config.js

```diff
@@ -3,6 +3,9 @@ import { faker } from '@faker-js/faker';
 import { clear } from './dataBase';
 
 module.exports = defineConfig({
+  retries: {
+    openMode:5
```

Comment:
There is a missing space after the colon in 'openMode:5'. It should be 'openMode: 5' for better readability.